tests/test_tools.py

```diff
@@ -37,3 +41,22 @@ def mock_get(limit=100, offset=0):
         notes_iterator = openreview.tools.iterget(self.client.get_notes)
         notes_list = list(notes_iterator)
         assert notes_list is not None, "Notes iterator failed"
+
+    def test_iterget_notes(self):
+        batch_size = 1000
+        notes_iterator = openreview.tools.iterget_notes(self.client, None, batch_size=batch_size)
```

Comment:
I'm not sure about passing the batch_size in the function, this should be an internal setting.

Reply:
Can you do?

```
openreview.tools.iterget_notes(self.client)
```

With no  invitation parameter

Reply:
1. We can make batch_size internal if you want, but it is optional right now, so the user doesn't have to touch it if they don't want. I just set the variable in the tests to be explicit. But if you think it's better to not have it available at all then I can take it out.

2. You can't call the iterget_notes function without the invitation, but I think you're right that you should be able to. I'll remove the invitation argument so that this is possible (which will mean that any invitation setting will be passed through the **params argument), but it'll break some old code

Reply:
This might be slightly off topic, but what do you guys think about some documentation explaining how to use **params for "iterget" ?

Reply:
I can add some explanatory comments. **kwargs is a feature of the language, so I don't think we should explain it in too much depth, but I agree it could use some comments

Reply:
**params can not be any value, it should be only the parameters accepted in the get_references call, may be add a link to the doc of the get_references.

Reply:
ah, right. so maybe the iterget_notes/references/tags functions shouldn't have **params, and should instead have the same arguments as their non-iterator equivalents.

iterget should still have **params though, in my opinion.

I'm going to come back to this after I finish setting up dummy sites for ICLR and NIPS, so melisa and zach can test